services/platform/app/components/ui/data-display/zoom-pan-viewer.tsx

```
@@ -0,0 +1,166 @@
+        isBottom
+          ? 'bg-background text-foreground flex items-center gap-2 rounded-full px-4 py-2 shadow-xl ring-1 ring-white/10'
+          : 'bg-muted flex items-center gap-1 rounded-lg p-1',
+        toolbarClassName,
+      )}
+    >
+      <button
+        onClick={zoomOut}
+        disabled={!canZoomOut}
+        className={buttonClass}
+        aria-label={t('imagePreview.zoomOut')}
+      >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)